gene-network/src/main/java/org/molgenis/ui/genenetwork/matrix/meta/MatrixMetadata.java

```diff
@@ -47,7 +47,7 @@ public void init()
 		setLabel("Matrix metadata");
 		setDescription("metadata with information about the matrix file");
 		addAttribute(ID, ROLE_ID);
-		addAttribute(FILE_LOCATION, ROLE_LABEL, ROLE_LOOKUP).setLabel("Location of the matrix file").setUnique(true)
```

Comment:
You might want to use the matrix with different (or without) row and column mappings.